include/openssl/core_numbers.h

```diff
@@ -413,6 +415,8 @@ OSSL_CORE_MAKE_FUNC(int, OP_keyexch_derive, (void *ctx,  unsigned char *secret,
 OSSL_CORE_MAKE_FUNC(int, OP_keyexch_set_peer, (void *ctx, void *provkey))
 OSSL_CORE_MAKE_FUNC(void, OP_keyexch_freectx, (void *ctx))
 OSSL_CORE_MAKE_FUNC(void *, OP_keyexch_dupctx, (void *ctx))
+OSSL_CORE_MAKE_FUNC(int, OP_keyexch_get_params, (const OSSL_PARAM params[]))
```

Comment:
It should be an array of writeable OSSL_PARAM objects, to return the value to the caller.

Reply:
Fixed